packages/run-protocol/test/reserve/test-reserve.js

```diff
@@ -338,3 +339,55 @@ test('request more collateral than available', async t => {
     'expecting more',
   );
 });
+
+test('reserve track shortfall', async t => {
+  /** @param {NatValue} value */
+  const electorateTerms = { committeeName: 'EnBancPanel', committeeSize: 3 };
+  const timer = buildManualTimer(console.log);
+
+  const { reserve, space, zoe } = await setupReserveServices(
+    t,
+    electorateTerms,
+    timer,
+  );
+
+  const runBrand = await space.brand.consume.RUN;
+
+  const shortfallReporterSeat = await E(zoe).offer(
+    E(reserve.reserveCreatorFacet).getShortfallReportInvitation(),
+  );
+  const reporterFacet = await E(shortfallReporterSeat).getOfferResult();
+
+  await E(reporterFacet).addLiquidationShortfall(
+    AmountMath.make(runBrand, 1000n),
+  );
+  const metricsSub = await E(reserve.reserveCreatorFacet).getMetrics();
+  const m = await subscriptionTracker(t, metricsSub);
+  await m.assertInitial({
+    allocations: {},
+    shortfall: AmountMath.makeEmpty(runBrand),
+  });
+  await m.assertChange({
+    shortfall: { value: 1000n },
```

Comment:
nice and clean expressing what has changed.

please keep a running value and adjust it. e.g.,

```
  await E(reporterFacet).addLiquidationShortfall(
    AmountMath.make(runBrand, 500n),
  );
  totalShortfall += 500n;
  await m.assertChange({
    shortfall: { value: totalShortfall },
  });
```

Reply:
done